docs/reference/getting-started.asciidoc

```diff
@@ -736,7 +465,10 @@ to clutter the docs with it:
     "failed" : 0
```

Comment:
I believe the changes in this section are breaking the CI build. Left a few suggestions to fix.